services/wallet/responses/router_suggested_routes.go

```diff
@@ -7,8 +7,7 @@ import (
 
 type RouterSuggestedRoutes struct {
 	Uuid          string                `json:"Uuid"`
-	Best          routes.Route          `json:"Best,omitempty"`
-	Candidates    routes.Route          `json:"Candidates,omitempty"`
```

Comment:
won't we still want to have multiple routes for different Swap/Bridge providers? Most aggregators will select the best one by default but let you check/choose the others.